Content.Client/Lobby/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -1037,7 +1070,7 @@ private void OpenLoadout(JobPrototype? jobProto, RoleLoadout roleLoadout, RoleLo
 
             _loadoutWindow = new LoadoutWindow(Profile, roleLoadout, roleLoadoutProto, _playerManager.LocalSession, collection)
             {
-                Title = jobProto?.ID + "-loadout",
+                Title = jobProto?.ID + " loadout",  //Moffstation - replaced preceding dash with a space, antag roles have a null jobproto and this makes that more subtle on the loadout screen
```

Comment:
```suggestion
                // Moffstation Start
                Title = jobProto?.ID switch
                {
                    null => "loadout",
                    var id => $"{id}-loadout",
                },
                // Moffstation End
```
Just explicitly handle null cases.